code/drasil-data/Data/Drasil/Citations.hs

```diff
@@ -30,18 +30,18 @@ koothoor2013 =
 parnas1972 = cArticle [dParnas]
   "On the Criteria To Be Used in Decomposing Systems into Modules"
   "Communications of the ACM" 1972
-  [pages [1053, 1058]] "dParnas1972"
+  [pages [1053..1058]] "dParnas1972"
```

Comment:
as far as I know `[1053..1058]` in Haskell expands to `[1053,1054,1055,1056,1057,1058]`. Is this really what you want?  Because how will you tell that apart from the specification "page 2, 5-6" ?

Reply:
Yes, this is desired. The `numList` function in `utils` would parse [2, 5, 6] into ["2", "5-6"], and the above expanded list into ["1053-1058"].

Reply:
kind of hacky, but acceptable.